modules/logging-svc/src/infrastructure/es_log_storage.ts

```diff
@@ -49,11 +49,16 @@ export class ElasticsearchLogStorage implements ILogStorageAdapter {
     }
 
     async init(): Promise<void> {
-        this._logger.info("ElasticsearchLogStorage initialised");
+        try{
+            this._logger.info("ElasticsearchLogStorage initialised");
 
-        // test the connection
-        const info = await this._client.info();
-        this._logger.info(`Connected to elasticsearch instance with name: ${info.name}, and cluster name: ${info.cluster_name}`);
+            // test the connection
+            // const info = await this._client.info();
+            // this._logger.info(`Connected to elasticsearch instance with name: ${info.name}, and cluster name: ${info.cluster_name}`);
+        }
+        catch (err) {
```

Comment:
Why are catching the error here? Would it not be better to fail rather?

Reply:
When this connection checking code executes, the event logging into elasticsearch is not working. I added this to check that issue. i will remove it .

Reply:
We still need to check the connection, this change removed that.
Either we have the check inside a try/catch to log and re-throw, or we don't have a try/catch and let it fail. We must never allow it to fail silently like this.

Reply:
I removed that try/catch

Reply:
It's still there, failing silently